native/cocos/renderer/frame-graph/ImmutableState.cpp

```diff
@@ -27,7 +27,7 @@
 #include "ImmutableState.h"
 #include <vector>
 #include "DevicePassResourceTable.h"
-#include "gfx-base/GFXDef-common.h"
+#include "gfx-base/GFXBarrier.h"
```

Comment:
Barrier related functions are moved to the GFXBarrier.h. These implementations will be reused in RenderGraph execution.